### PR TITLE
chore(README): Update dev instructions for Arch Linux users

### PR DESCRIPTION
**Pull request by rathmerdominik**
I wanted to start work on implementing OpenID Connect and got nasty errors when trying to install the dev setup on Arch Linux.

Here is what helped :smile:

### CODE REVIEW

File: DEVELOPER_SETUP.md [hunk 1]
```diff
@@ -48,6 +48,13 @@ Then create the virtual environment
 poetry install --sync
 ```
 
+If you are on Archlinux you need to run the command as follows:
+
+```sh
+# https://bbs.archlinux.org/viewtopic.php?id=296542
```
Comment (zurdi15) [inline DEVELOPER_SETUP.md:54]: The info link is duplicated

Comment (adamantike) [reply DEVELOPER_SETUP.md:54]: We could also mention the open PR that, when merged, will allow us to remove this specific instruction:
https://github.com/mariadb-corporation/mariadb-connector-python/pull/29

File: DEVELOPER_SETUP.md [hunk 2]
```diff
@@ -48,6 +48,13 @@ Then create the virtual environment
 poetry install --sync
 ```
 
+If you are on Archlinux you need to run the command as follows:
```
Comment (adamantike) [inline DEVELOPER_SETUP.md]:
```suggestion
If you are on Arch Linux or another Arch-based distro, you need to run the command as follows:
```

**Comment by gantoine** [comment]
@rathmerdominik That's awesome! There's a work-in-progress branch you can use as a base, or get inspired from: https://github.com/rommapp/romm/tree/openid-connect
There's no active work on that branch right now so the feature's all yours.

File: DEVELOPER_SETUP.md (hunk 1 above)
Comment (zurdi15) [reply DEVELOPER_SETUP.md:54]: Totally agree!

Comment (rathmerdominik) [reply DEVELOPER_SETUP.md:54]: oh. My bad
I forgot to remove it from behind the instruction :laughing: 
Will remove real quick

**Review by gantoine — APPROVED** [review]

**Comment by gantoine** [comment]
Thanks for the PR! Join the discord if you haven't done so, we can chat in #open-development about this and other features.

**Pull request merged**